cypress/e2e/seededFlows/notificationsFlows/notificationInitialization.spec.js

```diff
@@ -38,5 +38,11 @@ describe('Notification initialization', () => {
       cy.get('.reaction-button').should('exist');
       cy.get('.reacted').should('exist');
     });
+
+    it('initializes comment subscriptions', () => {
+      // FeatureFlag makes this tricky to test with Cypress
+      // TODO: Uncomment this as part of cleanup for FeatureFlag :subscribe_to_comments
+      // cy.get('.subscribe-button').should('exist');
```

Comment:
Fwiw, in 494d5bc, I set things up so that _this specific feature flag_ is always true for Cypress. For now, this works (this feature is generally additive - we need to test the new behavior alone, we aren't needing to confirm that old & new paths continue working) and we can figure out something better when we need to.